sunpy/map/mapbase.py

```diff
@@ -2316,7 +2316,7 @@ def _process_levels_arg(self, levels):
             raise u.UnitsError("This map has no unit, so levels can only be specified in percent "
                                "or in u.dimensionless_unscaled units.")
 
-    def draw_contours(self, levels, axes=None, **contour_args):
+    def draw_contours(self, levels, axes=None, fill=False, **contour_args):
```

Comment:
```suggestion
    def draw_contours(self, levels, axes=None, *, fill=False, **contour_args):
```
Let's not add a positional keyword argument